static/sass/styles.scss

```
@@ -400,6 +400,48 @@ summary {
   }
 }
 
+.pro-card-container {
+  grid-gap: 0;
+  .pro-card {
+    padding: calc(#{$spv--large} - 1px) calc(#{$spv--large} - 1px)
+      $spv--strip-deep * 0.5;
+    @media (min-width: $breakpoint-small) {
+      padding: calc(#{$spv--small} - 1px) calc(#{$spv--large} - 1px)
+        $spv--strip-deep;
+    }
+  }
+}
+.bg-light {
+  background-color: #f7f7f7;
+}
+
+.bg-mid-dark {
+  background-color: #666;
+  color: white;
+  hr {
+    background-color: rgba(217, 217, 217, 0.2);
```

again, pleaase use the corrct variable 
```suggestion
    background-color: $colors--dark-theme--border-low-contrast;
```